cms/sitemaps/cms_sitemap.py

```diff
@@ -45,13 +45,17 @@ def items(self):
         return all_pages
 
     def lastmod(self, page):
-        from cms.models import Placeholder, CMSPlugin
-        placeholders_qs = Placeholder.objects.filter(page=page)
-        placeholders_ids = list(placeholders_qs.values_list('id', flat=True))
-        plugins_qs = CMSPlugin.objects.filter(placeholder__in=placeholders_ids)
-        latest_plg_mod_date = plugins_qs.order_by(
-            '-changed_date').values_list('changed_date', flat=True)[:1]
         mod_dates = [page.changed_date, page.publication_date]
+        latest_plg_mod_date = None
+        with connection.cursor() as cursor:
+            cursor.execute(
+                "select cmsplg.changed_date from cms_cmsplugin cmsplg " +
+                "inner join cms_page_placeholders cmsp " +
+                "on cmsp.id=cmsplg.placeholder_id " +
+                "where cmsp.page_id = %s " +
+                "order by cmsplg.changed_date desc limit 1", [page.id])
+            latest_plg_mod_date = cursor.fetchone()
+
```

Comment:
I did some quick testing. It's MySQL, so performance is all over the map, but I'm seeing them as basically identical. Sometimes they take 2ms and then when you re-run it, it goes down to .9ms, but in either case, it should be fast enough. If you add EXPLAIN, they also seem about the same. I think we should use the ORM.